catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/OperationsStorageSupport.java

```diff
@@ -44,22 +44,20 @@
   private static final Logger INGEST_LOGGER = LoggerFactory.getLogger(Constants.INGEST_LOGGER_NAME);
 
   //
-  // Injected propertiers
+  // Injected properties
   //
   private final SourceOperations sourceOperations;
 
   private final QueryOperations queryOperations;
 
-  private Historian historian;
-
   public OperationsStorageSupport(
       SourceOperations sourceOperations, QueryOperations queryOperations) {
     this.sourceOperations = sourceOperations;
     this.queryOperations = queryOperations;
   }
 
   public void setHistorian(Historian historian) {
```

Comment:
❓ Does this method need to exist at all anymore?

Reply:
great question. No, I found that it lead to a lot to other changes like test classes. Tried to address those as well.

Reply:
removed the method and the domino effect of code that could be removed as a result.